rs/tests/ict/cmd/testnetCreateCmd.go

```diff
@@ -183,6 +183,21 @@ func TestnetCommand(cfg *TestnetConfig) func(cmd *cobra.Command, args []string)
 		}
 		command := []string{"bazel", "test", target, "--config=systest"}
 		command = append(command, "--cache_test_results=no")
+		icDashboardsDir, err := sparse_checkout("git@github.com:dfinity-ops/k8s.git", "", []string{"bases/apps/ic-dashboards"})
+		if err != nil {
+			cmd.PrintErrln(YELLOW + "Failed to sync k8s dashboards. Received the following error: " + err.Error())
+		} else {
+			cmd.PrintErrln(GREEN + "Successfully synced dashboards to path " + icDashboardsDir)
+			icDashboardsDir = filepath.Join(icDashboardsDir, "bases", "apps", "ic-dashboards")
+			cmd.Println(GREEN + "Will use " + icDashboardsDir + " as a root for dashboards")
+
+			if err := replace_in_directory(icDashboardsDir, "${datasource}", "000000001"); err != nil {
```

Comment:
can you explain what this does and why?

Reply:
The thing is that in k8s repo setup, we have prometheus configured as a variable, and in farm testnets we have the uid set to `000000001`. So what this does is finds all the mentions of `${datasource}` and replaces is it with `000000001` so that dashboards would use correct prometheus datasource to pull data.

Reply:
@NikolaMilosa would it be possible to configure Grafana such that `${datasource}` resolves to the prometheus datasource such that we don't need to do this replacement?

Note we define the prometheus datasource in Grafana on the PrometheusVM in:
https://github.com/dfinity-lab/farm/blob/355c4c240c657c8e753ac64bccf478105ab76b6e/universal-vm/configuration-prometheus.nix#L88-L98

Is there something I can configure in Grafana to make this work?